chartfx-dataset/src/main/java/de/gsi/dataset/serializer/spi/iobuffer/FieldCollectionsHelper.java

```diff
@@ -0,0 +1,62 @@
+package de.gsi.dataset.serializer.spi.iobuffer;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.Queue;
+import java.util.Set;
+
+import de.gsi.dataset.serializer.FieldSerialiser;
+import de.gsi.dataset.serializer.IoClassSerialiser;
+import de.gsi.dataset.serializer.utils.ClassUtils;
+
+public class FieldCollectionsHelper {
+    private FieldCollectionsHelper() {
+        // utility class
+    }
+
+    /**
+     * registers default Collection, List, Set, and Queue interface and related helper methods
+     *
+     * @param serialiser for which the field serialisers should be registered
+     */
+    public static void register(final IoClassSerialiser serialiser) {
```

Comment:
Codacy found an issue: [The method 'register(IoClassSerialiser)' has an NPath complexity of 256, current threshold is 200](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5905529)